bin/node-template/node/src/command.rs

```diff
@@ -58,6 +58,10 @@ impl SubstrateCli for Cli {
 			)?),
 		})
 	}
+
+	fn runtime_version() -> &'static RuntimeVersion {
+		&node_template_runtime::VERSION
```

Comment:
THis only returns the native version. So the function name should be changed to reflect this.

Reply:
Sorry, I don't understand what you mean. Could you go into more detail?

Reply:
(He means you should rename the function to "runtime_native_version" or something I think)